module/README.md

```diff
@@ -1,5 +1,8 @@
 This kernel module implements a basic interface to the IVSHMEM device for
-LookingGlass when using LookingGlass in VM->VM mode.
+LookingGlass when using LookingGlass in VM→VM mode.
```

Comment:
'→' Please revert to the hyphen, no need for this change or extended ASCII characters.

Reply:
Reverted.